src/azure/azureController.ts

```diff
@@ -239,6 +249,35 @@ export class AzureController {
 		}
 	}
 
+	/**
+	 * Returns Azure subscriptions with tenant and token for each given account
+	 */
+	public async getAccountSessions(account: IAccount): Promise<mssql.IAzureAccountSession[]> {
```

Comment:
Why is this here? You already have a subscription client in the azureResourceController so it seems better to just use that one

Reply:
if you are asking about why the function is here, I thought it's related to account and that's why I put it here. I was debating about having this function in this API or resources API and I was not sure at the end. as you see, in extension APIs, I put this in account service not resource service. a=if you think it make more sense to put it in resources, I can move it